python/taichi/lang/misc.py

```diff
@@ -346,6 +346,9 @@ def init(arch=None,
     # Check version for users every 7 days if not disabled by users.
     _version_check.start_version_check_thread()
 
+    # save current working directory since it may be changed by the vulkan backend
```

Comment:
```suggestion
    # Save the current working directory since it may be changed by the Vulkan backend initialization on OS X (https://github.com/taichi-dev/taichi/issues/4811)
```